### DISCUSSION TITLE
Smart Start > DSK values deleted with 9.10.2 update?

### DISCUSSION THREAD

**Discussion by drowl87** [zwave-js/zwave-js-ui#3658] [Q&A]
I just updated to the latest image > running on a Synology NAS and noticed that all of my smart start/dsk entries are gone! Has anyone else had that happen or know how to somehow restore them? I downgraded the image back to 9.9, but the whole screen is blank still.

**Comment by robertsLando** [comment]
@drowl87 how are you running zui? If with docker could you provide the doceker-compose file? Ensure the volume is setup correctly, the dsk informations are stored in a json in sotore folder.

@AlCalzone any clue if last update could cause this?

**Reply by robertsLando** [reply]
Seems there is already an open issue related to this, follow updates there: https://github.com/zwave-js/node-zwave-js/issues/6755
